### PR TITLE
feat(docs): emit/event docs 

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR adds docs for emit/event. It adds reference docs, and a section in effective Gno. More practical examples will be added to Gno By Example.

Closes: #2003 

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

**Comment by leohhhn** [comment]
Hey @zivkovicmilos,

> I find it strange that you didn't mention the required PR on the docs repo anywhere:

I've linked this PR in the docs.gno.land repo PR, so the connection should be visible. 

> Please fix the failing PR on that repo, it can't even build because of invalid IDs.

Due to the setup that we currently have with the documentation infra, it is currently not possible to build or preview the documentation on the docs.gno.land repo - which is why the CI is red on the mentioned PR. Specifically, this is because the CI on the other repo fetches the monorepo `master` branch and tries to build the files, but since this PR here has not been merged yet (and thus the files are missing, hence the 'invalid ID' error), the build will fail. 

Currently, the only way to get the CI fixed on the other repo is to merge content here first. This PR is ready for review.

File: docs/concepts/stdlibs/events.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ]
+}
+```
+
+You can fetch the ABCI response of a specific block by using the `/block_results` 
+RPC endpoint. 
+
+
+
+
```
Comment (notJoon) [inline docs/concepts/stdlibs/events.md:60]:
```suggestion
You can fetch the ABCI response of a specific block by using the `/block_results` 
RPC endpoint.

```

niptick. just for linting rules

File: docs/concepts/stdlibs/events.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Read more about events & `Emit()` in 
+[Effective Gno](../effective-gno.md#emit-gno-events-to-make-life-off-chain-easier),
+and the `Emit()` reference [here](../../reference/stdlibs/std/chain.md#emit).
+
+## Data contained in a Gno Event
+
+An event contained in an ABCI response of a block will include the following
+data:
+
+```
```
Comment (notJoon) [inline docs/concepts/stdlibs/events.md:36]:
```suggestion
``` json
```

For consistency, it would be good to specify the JSON here.

**Review by notJoon — APPROVED** [review]
I left some minor things, but overall it's well written. 👍

File: docs/concepts/effective-gno.md [hunk 3]
```diff
@@ -425,7 +492,7 @@ The goal is usually to store the admin address or a list of addresses
 owners. These helper functions should check if the caller of a function is
 whitelisted or not.
 
-Let's deep dive into the different access control mechanisms we can use:
+Lets deep dive into the different access control mechanisms we can use:
```
Comment (zivkovicmilos) [inline docs/concepts/effective-gno.md:495]: This is a typo, the correct form was `Let's`

**Review by zivkovicmilos — APPROVED** [review]

File: docs/concepts/effective-gno.md (hunk 3 above)
Comment (leohhhn) [reply docs/concepts/effective-gno.md]: Ah, you're right, don't know what was going through my head. Fixed it: [3091d76](https://github.com/gnolang/gno/pull/2047/commits/3091d76e2c667d9bbd300a0af5cc539f87d8a9df)

**Pull request merged**